content/es/style-guide/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Guia de estilos
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-ruler-horizontal'></i>
+---
+
+Esta guia de estilos te ayudará a entender la audiencia del Glosario, la estructura de las definiciones, el nivel de detalle requerido, y como mantener un estilo consistente.
+
+El Glosario Cloud Native sigue las [guias de estilo por defecto](https://github.com/cncf/foundation/blob/master/style-guide.md) del repositorio de CNCF. Adicionalmente, sigue las siguientes reglas:
+
+1. Utilice un lenguaje sencillo y accesible, evitando la jerga técnica y palabras de moda.
+2. [Evite el lenguaje coloquial](https://en.wikipedia.org/wiki/Colloquialism)
+3. [Use un lenguaje literal y concreto](http://guidetogrammar.org/grammar/composition/abstract.htm)
+4. [Omite contracciones](https://en.wikipedia.org/wiki/Contraction_(grammar))
+5. [Use la voz pasiva con moderación] (https://www.ef.com/ca/english-resources/english-grammar/passive-voice/)
+6. [Trate de formular declaraciones en forma positiva] (https://examples.yourdictionary.com/positive-sentence-examples.html)
+7. [Sin signos de exclamación fuera de las comillas] (https://www.grammarly.com/blog/exclamation-mark/)
```

Comment:
To make links work:

```suggestion
5. [Use la voz pasiva con moderación](https://www.ef.com/ca/english-resources/english-grammar/passive-voice/)
6. [Trate de formular declaraciones en forma positiva](https://examples.yourdictionary.com/positive-sentence-examples.html)
7. [Sin signos de exclamación fuera de las comillas](https://www.grammarly.com/blog/exclamation-mark/)
```

Reply:
fix on 4190b0989dcd8d1684da424859323f1da1c9abf2